runtime/syntax/tex.vim

```diff
@@ -745,6 +755,8 @@ if has("conceal") && &enc == 'utf-8'
     \ ['lceil'		, '⌈'],
     \ ['ldots'		, '…'],
     \ ['le'		, '≤'],
+    \ ['left|'		, '|'],
+    \ ['left\|'		, '‖'],
     \ ['left('		, '('],
     \ ['left\['		, '['],
```

Comment:
And while I'm at it: this should probably be `\left[` (without the backslash), see the corresponding entry for `\right]`.

(@leun4m not your fault, but I'm surprised nobody caught that before!)